<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetEditStage</class>
 <widget class="QWidget" name="WidgetEditStage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="labelTimeLimit">
     <property name="text">
      <string>Time Limit [s]:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinBoxTimeLimit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="spinBoxTimeLimit">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>3600.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBoxMessages">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Messages</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonMessagesAdd">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonMessagesRemove">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButtonMessagesEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QListWidget" name="listWidgetMessages"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBoxInitUnits">
     <property name="title">
      <string>Init Units</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonInitUnitAdd">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButtonInitUnitRemove">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QListWidget" name="listWidgetInitUnits"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBoxObjectives">
     <property name="title">
      <string>Objectives</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonObjAdd">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButtonObjEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonObjRemove">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QListWidget" name="listWidgetObjectives"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="pushButtonDone">
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelStageName">
     <property name="text">
      <string>Name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>lineEditStageName</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditStageName"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditStageName</tabstop>
  <tabstop>spinBoxTimeLimit</tabstop>
  <tabstop>pushButtonMessagesAdd</tabstop>
  <tabstop>pushButtonMessagesEdit</tabstop>
  <tabstop>pushButtonMessagesRemove</tabstop>
  <tabstop>listWidgetMessages</tabstop>
  <tabstop>pushButtonInitUnitAdd</tabstop>
  <tabstop>pushButtonInitUnitRemove</tabstop>
  <tabstop>listWidgetInitUnits</tabstop>
  <tabstop>pushButtonObjAdd</tabstop>
  <tabstop>pushButtonObjEdit</tabstop>
  <tabstop>pushButtonObjRemove</tabstop>
  <tabstop>listWidgetObjectives</tabstop>
  <tabstop>pushButtonDone</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
